### PR TITLE
Manifest files for network performance measurement

### PR DESCRIPTION
**Pull request by hanamantagoudvk**
It includes:

* CRD yaml
* Role binding yaml
* NetworkTestRequest CR template yaml
* Worker Pod deployment yaml

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/network/manifests/networktestrequests.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+spec:
+  clientPodName: {{.clientPodName}} 
+  serverPodName : {{.serverPodName}}
+  serverPodIP: {{.serverPodIP}}
+  protocol: {{.protocol}}
+  duration: {{.duration}}
+  clientStartTimestamp: {{.clientStartTimestamp}}
+  numberOfClients: {{.numberOfClients}}
+status:
+  metrics: []
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/network/manifests/networktestrequests.yaml:18]: add EOL

File: clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      containers:
+      - image:  vivekthrivikraman/netperfbenchmark:0.1
+        name: worker
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+#        resources:
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml:33]: Please cleanup this PR.

Comment (hanamantagoudvk) [reply clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml]: done

File: clusterloader2/pkg/measurement/common/network/manifests/networktestrequests.yaml (hunk 1 above)
Comment (hanamantagoudvk) [reply clusterloader2/pkg/measurement/common/network/manifests/networktestrequests.yaml]: done

File: clusterloader2/pkg/measurement/common/network/manifests/roleBinding.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: networktestrequests-rbac
+subjects:
+  - kind: ServiceAccount
+    # Reference to upper's `metadata.name`
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/network/manifests/roleBinding.yaml:7]: remove these comments

File: clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        group: network
+        name: worker
+    spec:
+      containers:
+      - image:  vivekthrivikraman/netperfbenchmark:0.1
+        name: worker
+        imagePullPolicy: IfNotPresent
+        resources:
+          limits:
+            cpu: "1"
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml:25]: let's set limits and requests to the same value - that will make it more predictible

Comment (hanamantagoudvk) [reply clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml:25]: ok done.

File: clusterloader2/pkg/measurement/common/network/manifests/roleBinding.yaml (hunk 3 above)
Comment (hanamantagoudvk) [reply clusterloader2/pkg/measurement/common/network/manifests/roleBinding.yaml]: ok done

**Pull request merged**